src/utils/env.ts

```diff
@@ -21,7 +21,7 @@ export function isAppUniswapOrg({ hostname }: { hostname: string }): boolean {
 
 export function isSentryEnabled(): boolean {
   // Disable in e2e test environments
-  if (isStagingEnv() || (isProductionEnv() && !isAppUniswapOrg(window.location))) return false
```

Comment:
I'm thinking this is necessary... No point in updating Sentry environment to staging if we're not actually going to do any reporting??

Reply:
yes, I just wonder where else this is being set. Probably in our vercel builds, which we do _not_ want reporting to sentry. I think we may want an additional check on window.location, but that can be done in a follow-up.

Reply:
Could you change this to `(isStagingEnv() && !isAppUniswapStagingOrg(window.location))`? That way we'll only send sentry events from the staging build.

Reply:
Sounds good :)